src/makeline-service/README.md

```diff
@@ -4,34 +4,125 @@ This is a Golang app that provides an API for processing orders. It is meant to
 
 It is a simple REST API written with the Gin framework that allows you to process orders from a RabbitMQ queue and send them to a MongoDB database.
 
-## Running the app locally
-
-### Prerequisites
+## Prerequisites
 
 - [Go](https://golang.org/doc/install)
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [MongoSH](https://docs.mongodb.com/mongodb-shell/install/)
 
-### Running the app
+## Message queue options
 
-The app relies on RabbitMQ and MongoDB. Additionally, to simulate orders, you will need to run the [order-service](../order-service) with the [virtual-customer](../virtual-customer) app. A docker-compose file is provided to make this easy.
+This app can connect to either RabbitMQ or Azure Service Bus using AMQP 1.0. To connect to either of these services, you will need to provide appropriate environment variables for connecting to the message queue.
 
-To run the necessary services, clone the repo, open a terminal, and navigate to the `makeline-service` directory. Then run the following command:
+### Option 1: RabbitMQ
+
+To run this against RabbitMQ. A docker-compose file is provided to make this easy. This will run RabbitMQ, the RabbitMQ Management UI, and enable the `rabbitmq_amqp1_0` plugin. The plugin is necessary to connect to RabbitMQ using AMQP 1.0.
+
+With the services running, open a new terminal and navigate to the `makeline-service` directory.
+
+Set the connection information for the RabbitMQ queue by running the following commands to set the environment variables:
 
 ```bash
-docker compose up
+export ORDER_QUEUE_URI=amqp://localhost
+export ORDER_QUEUE_USERNAME=username
+export ORDER_QUEUE_PASSWORD=password
+export ORDER_QUEUE_NAME=orders
+```
+
+### Option 2: Azure Service Bus
+
+To run this against Azure Service Bus, you will need to create a Service Bus namespace and a queue. You can do this using the Azure CLI. 
+
+```bash
+az group create --name <resource-group-name> --location <location>
+az servicebus namespace create --name <namespace-name> --resource-group <resource-group-name>
+az servicebus queue create --name orders --namespace-name <namespace-name> --resource-group <resource-group-name>
+```
+
+Once you have created the Service Bus namespace and queue, you will need to create a shared access policy with the **Listen** permission for the namespace.
+
+```bash
+az servicebus namespace authorization-rule create --name listener --namespace-name <namespace-name> --resource-group <resource-group-name> --queue-name orders --rights Listen
```

Comment:
I think `--queue-name` is here by mistake. I removed it and the command worked.

Reply:
Removed the erroneous flag.